src/actions/wallet/L1/writeSendMessage.ts

```diff
@@ -18,14 +19,14 @@ export type WriteSendMessageParameters<
   TChain extends Chain | undefined = Chain,
   TAccount extends Account | undefined = Account | undefined,
   TChainOverride extends Chain | undefined = Chain | undefined,
+  _chainId = TChain extends Chain ? TChain['id'] : number,
 > =
-  & { args: SendMessageParameters }
+  & { args: SendMessageParameters; l1CrossDomainMessenger: RawOrContractAddress<_chainId> }
   & L1WriteActionBaseType<
```

Comment:
Is it even useful to have this type, now? Should we just use WriteContractParameters or SendTransactionParameters?